pkg/controller/volume/attachdetach/attach_detach_controller.go

```diff
@@ -649,6 +667,65 @@ func (adc *attachDetachController) processVolumesInUse(
 	}
 }
 
+// installCRDs creates the specified CustomResourceDefinition for the CSIDrivers object.
+func (adc *attachDetachController) installCRDs() error {
+	crd := &apiextensionsv1beta1.CustomResourceDefinition{
```

Comment:
I think the definition files should be stored in yaml format in a file. I don't think it's a good idea to construct them at runtime, it's too easy to make a change to the code and not realize you're introducing a compatibility problem.

Reply:
Noted. When we move this to a more permanent solution in beta (see https://github.com/kubernetes/kubernetes/pull/67803#discussion_r214497103), will consider this as well.

Reply:
> I think the definition files should be stored in yaml format in a file. I don't think it's a good idea to construct them at runtime, it's too easy to make a change to the code and not realize you're introducing a compatibility problem.

Why was this comment not considered necessary to address? I am somewhat dismayed by the attitude (echoed in the follow up PR) that the issues raised here don't matter for alpha.